scripts/07_generate_consensus_vectors.py

```
@@ -0,0 +1,400 @@
+        existing = Path(args.existing_dir)
+        for sub in sorted(existing.glob("*complex_scenarios")):
+            out_sub = out / sub.name
+            out_sub.mkdir(exist_ok=True)
+            for jf in sorted(sub.glob("*.json")):
+                old = json.loads(jf.read_text())
+                if "patterns" not in old:
+                    continue
+                new_patterns = {}
+                for nm, pd in old["patterns"].items():
+                    core = nm.split(" [rot:")[0]
+                    path = ["START"] + core.split(" -> ") + ["END"]
```

Ruff (0.15.20)

[warning] Consider `["START", *core.split(" -> "), "END"]` instead of concatenation

Replace with `["START", *core.split(" -> "), "END"]`

(RUF005)